OpTestConfiguration.py

```diff
@@ -385,6 +385,17 @@ def get_parser():
     kernelcmdgroup.add_argument("--remove-kernel-args",
                                 help="Kernel commandline option to be removed",
                                 default="")
+    kdumpgroup = parser.add_argument_group("Kdump group",
+                                           "Ipaddress username and password of remote server")
+    kdumpgroup.add_argument("--server-ip",
+                           help="IP address of remote server where dump will be stored",
+                           default="")
+    kdumpgroup.add_argument("--server-pw",
+                           help="Password of remote server where dump will be stored",
+                           default="")
```

Comment:
Sorry for the new changes. Just realised now, let us not use genric server-ip and password it may disturb other test.

let the variable be dump_server_ip and dump_server_password. it will be unique and will not trouble other tests.